analyzers/src/SonarAnalyzer.Common/Helpers/DotWriter.cs

```diff
@@ -59,16 +59,6 @@ public void WriteNode(string id, string header, params string[] items)
             this.writer.WriteLine("]");
         }
 
-        internal void WriteEdge(string startId, string endId, string label)
```

Comment:
I suggest to keep this one, even when not currently used.
It was very useful during our Roslyn CFG investigation sprint (I didn't have to spent too much time figuring out how to do it). I used it a lot there.

Reply:
This whole class is relevant only for debug output. So it offers help when we need it.

Reply:
And it will actually most likely be needed when we reimplement the `CfgSerializer` for the new CFG.